src/realm-settings/ProfilesTab.tsx

```diff
@@ -65,20 +80,18 @@ export const ProfilesTab = () => {
     },
   });
 
-  function cellFormatter(name?: IFormatterValueType) {
-    return (
-      <>
-        <Link to={""} key={`link-${name}`}>
-          {name}
-        </Link>{" "}
-        {profiles?.globalProfiles?.length! > 0 && (
-          <Label key={`label-${name}`} color="blue">
-            {t("global")}
-          </Label>
-        )}
-      </>
-    );
-  }
+  const cellFormatter = (row: ClientProfile) => (
+    <Link to={""} key={`label-${row.name}`}>
+      {row.name} {""}
+      {row.global ? (
+        <Label key={`label-${row.global}`} color="blue">
+          {t("global")}
+        </Label>
+      ) : (
+        ""
+      )}
```

Comment:
Since the `Label` component is shown conditionally and not in a `.map()` I believe there is no need to provide a `key` prop here. Also since the else condition is rendering an empty string perhaps it can be omitted?

```suggestion
      {row.global && <Label color="blue">{t("global")}</Label>}
```

Reply:
like this idea - nice catch! ty